src/Integration.UnitTests/Persistence/SolutionBindingCredentialsLoaderTests.cs

```diff
@@ -87,15 +87,15 @@ public void Save_ServerUriIsNull_CredentialsNotSaved()
 
             testSubject.Save(credentials, null);
 
```

Comment:
We had a bug in the tests, wrong assertion so they always passed. This was caught by the new Moq version.